src/Sylius/Component/Grid/Definition/Filter.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Grid\Definition;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class Filter
+{
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var string
+     */
+    private $type;
+
+    /**
+     * @var string
+     */
+    private $label;
+
+    /**
+     * @var array
+     */
+    private $options = array();
+
+    /**
+     * @param string $name
+     * @param string $type
+     */
+    private function __construct($name, $type)
+    {
+        $this->name = $name;
+        $this->type = $type;
+
+        $this->label = $name;
```

Comment:
It's some kind of default value, which you can overwrite by `setLabel` if you want.